### ISSUE TITLE
PseudoClasses do not resolve correct

### ISSUE THREAD

**Issue by matsrietdijk** [rtfeldman/elm-css#136]
For example:

```
[ a 
    [ color (hex "#000000")
    , hover
        [ color (hex "#111111") ]
    , active
        [ color (hex "#222222") ]
    , disabled
        [ color (hex "#333333") ]
    ]
]
```
Results in:
```
a { color: #000000 }
a:hover { color: #111111 }
a:hover:active { color: #222222 }
a:hover:active:disabled { color: #333333 }
```

**Comment by matsrietdijk** [comment]
Also if adding statements after a pseudo class the generated css is not was expected:
```
[ a 
    [ hover
        [ backgroundColor (hex "#000000") ]
    , color (hex "#1111111")
    ]
]
```
Results in:
```
a:hover { 
    background-color: #000000;
    color: #111111
}
```

**Comment by rtfeldman** [comment]
Thank you for the concise, detailed explanations! That will definitely help.

**Comment by rtfeldman** [comment]
Fixed in https://github.com/rtfeldman/elm-css/pull/142 and https://github.com/rtfeldman/elm-css/issues/143 and released as 4.0.1.

Thanks @tolgap for the fixes, and @matsrietdijk for the excellent error report!
